test/test_cluster.py

```diff
@@ -4,8 +4,8 @@
 import os.path as op
 import inspect
 import pybedtools
-import seqcluster.libs.parameters
-import seqcluster.libs.logger as mylog
+import seqcluster.seqcluster.libs.parameters
```

Comment:
is this right? seqcluster twice, or it should be just seqcluster.libs.parameters?

it is failing in python 2.7 right now.